### PR TITLE
Fix material folder ui

### LINKED ISSUE #1975: Fix styling for materials
To add a space between the folder icon and the name and also the number of items in a folder in parenthesis after the folder.

### PR DESCRIPTION
**Pull request by nicholette**
Fixes #1975

### CODE REVIEW

**Comment by weiqingtoh** [comment]
:+1:  @nicholette can add a screen shot for this?

**Comment by nicholette** [comment]
Ok

screenshot 2017-02-04 15 15 21
screenshot 2017-02-04 15 15 39

File: app/views/course/material/folders/_folder.slim [hunk 1]
```diff
@@ -1,8 +1,9 @@
 = content_tag_for(:tr, folder)
   th
     = link_to course_material_folder_path(current_course, folder) do
-      = fa_icon 'folder-open'.freeze
+      => fa_icon 'folder-open'.freeze
       => format_inline_text(folder.name)
+      => '(' + format_inline_text(folder.material_count + folder.children_count) + ')'
```
Comment (fonglh) [inline app/views/course/material/folders/_folder.slim]: Does Ruby's string interpolation work here? I think it should.

`"(#{format_inline_text(folder.material_count + folder.children_count)})"`

Comment (nicholette) [inline app/views/course/material/folders/_folder.slim]: Yep, it does. Will update.

**Comment by fonglh** [comment]
Squash to 1 commit.

**Comment by nicholette** [comment]
@fonglh squashed! merge now?

**Pull request merged**